python/nano/src/bigdl/nano/deps/neural_compressor/pytorch/quantized_model.py

```diff
@@ -20,21 +20,31 @@
 from neural_compressor.utils.pytorch import load
 from neural_compressor.model.model import PyTorchModel
 from bigdl.nano.utils.log4Error import invalidInputError
-from bigdl.nano.pytorch.context_manager import BaseContextManager
+from bigdl.nano.pytorch.context_manager import generate_context_manager
 
 
 class PytorchQuantizedModel(AcceleratedLightningModule):
-    def __init__(self, model):
+    def __init__(self, model, thread_num=None):
         super().__init__(model.model)
         self.quantized = model
-        self.context_manager = BaseContextManager()
+        self.thread_num = thread_num
```

Comment:
shall we make `thread_num` part of `AcceleratedLightningModule`?

Reply:
Yeah that's definitely correct. We are having a messy inherit situation for inference classes. Some of the class does not call init of their mother class at all, which makes `AcceleratedLightningModule` only function as a dummy base class. I will record this under the refactor umbrella issue.